lib/quadquizaminos_web/live/contests_live.html.leex

```diff
@@ -12,7 +12,7 @@
                     <%= text_input f, :name %>
                    <%= error_tag f, :name %>
                   </div>
-                 <div phx-update="ignore" class="column">
+                 <div phx-update="ignore" id="contest_date" class="column">
```

Comment:
Whats the reason for adding `id="contest_date"`?

Reply:
Because l got an error which required setting an ID on the container when setting the phx-update to "ignore"